templates/api/regulations.xml.twig

```diff
@@ -42,11 +42,22 @@
                 {% for trafficRegulation in regulationOrder.trafficRegulations %}
                     {% set location = trafficRegulation.location %}
                     {% set vehicleConditions = trafficRegulation.vehicleConditions %}
+                    {% set speed = trafficRegulation.maxSpeed %}
                     <trafficRegulation>
                         <status>active</status>
-                        <typeOfRegulation xsi:type="AccessRestriction">
-                            <accessRestrictionType>noEntry</accessRestrictionType>
-                        </typeOfRegulation>
+                        {% if speed %}
```

Comment:
Pour être pointilleux, on pourrait mettre ici `trafficRegulation.type === "speedLimitation" and speed`.